src/main/webapp/assets/modules/talent-search/search-result/search-result.con.js

```diff
@@ -1,23 +1,48 @@
 techlooper.controller("tsSearchResultController",
   function ($scope, $timeout, tsMainService, tsSearchResultService, $routeParams, $http, jsonValue, $location) {
+    try {
+      var request = CryptoJS.enc.Base64.parse($routeParams.text).toString(CryptoJS.enc.Utf8);
+      request = JSON.parse(request);
+    }
+    catch(e){
+      $location.path("/");
+    }
+
+    $scope.search = {
+      talents: [],
+      pageIndex: -1,
+      busy: false,
+      total: 1,
+      nextPage: function () {
+        if (this.busy || this.talents.length === this.total) {
+          return;
+        }
+        this.busy = true;
+        ++this.pageIndex;
+        request.pageIndex = this.pageIndex;
+        $http.post(jsonValue.httpUri.searchTalent, JSON.stringify(request))
+          .success(function (data, status, headers, config) {
```

Comment:
'config' is defined but never used.<br>'headers' is defined but never used.<br>'status' is defined but never used.